### PR TITLE
Moved variables that are in sbuttons.less to src/_variables.less file and imported _variables.less in file sbuttons.less and updated readme according to the changes

### PR DESCRIPTION
**Pull request by kuldeep123kd**
Issue:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,17 +66,17 @@ And then you can import the CSS file found in `dist/sbuttons.css` or the LESS fi
 
 ### Modifying Button Colors
 
-To modify button colors, import `src/sbuttons.less` in your Less file, then make changes to the variables after the import.
+To modify button colors, import `src/_variables.less` in your Less file, then make changes to the variables after the import.
```
Comment (shahednasser) [inline README.md:69]: Don't make changes here

File: README.md [hunk 2]
```diff
@@ -66,17 +66,17 @@ And then you can import the CSS file found in `dist/sbuttons.css` or the LESS fi
...
 
 For example, to change the blue color to a different shade
 
 ```
-@import '/path/to/sbuttons.less';
+@import '/path/to/_variables.less';
```
Comment (shahednasser) [inline README.md:74]: Don't make changes here

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kuldeep123kd) [reply README.md:69]: should i revert back those changes?

Comment (shahednasser) [reply README.md:69]: yes

Comment (kuldeep123kd) [reply README.md:69]: but this was mentioned in issue that " You should also change the link in the README for sbuttons.less under Modifying Button Colors to the new variables.less file and change the text as well to "_variables.less" "

Comment (kuldeep123kd) [reply README.md:69]: oh got it

Comment (kuldeep123kd) [reply README.md]: I did the changes as you requested.

**Comment by kuldeep123kd** [comment]
I did the changes as you requested. what should i do now?

**Review by kuldeep123kd — COMMENTED** [review]
I did revert back the changes as you requested.

**Comment by shahednasser** [comment]
Great work thank you!

@all-contributors please add @kuldeep123kd for code

**Pull request merged**